server/data_server.c

```diff
@@ -434,18 +434,20 @@ static bool write_block_base64(
 }
 
 
+static void set_frame_header(void *buffer, uint32_t frame_length) {
+    /* Update the first four bytes of the buffer with a header followed by a
+        * frame byte count. */
```

Comment:
Is this a tab I see before me?  If not, this is a funny looking indentation error.

Please check for both errant tabs and trailing whitespace, which I don't seem to be able to see in this view.

Reply:
Have just added a plugin (Render Whitespace on GitHub, really hope it's safe), and this appears to just be dodgy layout!

Reply:
yup, too many spaces...